### DISCUSSION TITLE
Keep a Chrome Instance running instead of spawning a new one each time?

### DISCUSSION THREAD

**Discussion by mrcrmn** [spatie/browsershot#829] [Q&A]
Hi there,

I was wondering if it was possible to keep a single chrome instance running in the background to which Browsershot can connect to instead of spawning a new instance for each PDF.
When I queue a batch of jobs to generate PDFs, each one will take around 10s on our prod server and I am looking into ways to make that more efficient.

Is this what the `setRemoteInstance` option is for? And if yes, how would you use it?

**Comment by DirectorVector** [comment]
I was looking into changing from my private project to this until I saw your question here. This was the reason I made a custom project, and so I went ahead and updated it today and published it for our company. I made it public if you want to check it out. Its a node project though, but it's a pretty simple little api that works well. https://github.com/MrBLT/PuppeteerPdfGenerator
